src/utils/util.ts

```diff
@@ -0,0 +1,75 @@
+import { toast } from "react-toastify";
+import * as blake from "blakejs";
+import { bech32m } from "bech32";
+
+export const DECIMAL = 100000000000;
+
+export const encodeToHash = (data: string) => {
```

Comment:
I see that this function is used to calculate hash of a transaction. It's better to get it on rust side by putting it into TransactionResult as a separate field.